app/frontend/packs/administration.js

```diff
@@ -12,3 +12,4 @@ require('@rails/activestorage').start();
 import 'foundation';
 import '../application.scss';
 import '../components/header';
+import 'selectize/dist/js/selectize.min.js';
```

Comment:
note: this seemed required to avoid an initialize error in the console

it seems weird as `administrate` depends on `selectize-rails` but I couldn't get sprockets happy with importing from that so for now I've just explicitly added it as a dependency using the same version that `selectize-rails` is providing